draft-ietf-core-dns-over-coap.md

```diff
@@ -489,19 +504,19 @@ contexts.
 {{Section 2.6 of -core-corrclar}} goes into more detail on what needs to be done
 when those are resumed from a new endpoint.
 
-When using unencrypted CoAP (see {{sec:unencrypted-coap}}), setting the ID of a DNS message to 0 as
+When using unprotected CoAP (see {{sec:unprotected-coap}}), setting the ID of a DNS message to 0 as
 specified in {{sec:req-caching}} opens the DNS cache of a DoC client to cache poisoning attacks
 via response spoofing.
 This document requires an unpredictable CoAP token in each DoC query from the client when CoAP is
-not secured to mitigate such an attack over DoC (see {{sec:unencrypted-coap}}).
+not secured to mitigate such an attack over DoC (see {{sec:unprotected-coap}}).
 
-For encrypted usage with DTLS or OSCORE the impact of a fixed ID on security is limited, as both
+For confidential communication with DTLS or OSCORE the impact of a fixed ID on security is limited, as both
```

Comment:
s/with/via/?

(communication "with" could be read as about the peer -- "let me communicate with XYZ today")

Reply:
```suggestion
For confidential communication via DTLS or OSCORE the impact of a fixed ID on security is limited, as both
```